src/c_api/c_api.cc

```diff
@@ -195,6 +196,20 @@ int MXNDArrayCreateEx(const mx_uint *shape,
   API_END();
 }
 
+int MXNDArrayCreateEx(const mx_int64 *shape,
```

Comment:
Also you are directly changing this C API which might break other language bindings that use this C API.